recipes/pypeln/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "pypeln" %}
+{% set version = "0.4.4" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "c9a3b72f981441e0ea7576cf54c4cd77289b6a2322497da43581cf9b47999ff4"
+
+build:
+  number: 0
```

As this package isn't platform-specific, this can be `noarch`.
```suggestion
  number: 0
  noarch: python
```